flocker/node/agents/functional/test_gce.py

```diff
@@ -144,6 +169,64 @@ def test_multiple_cluster(self):
                          list(x.dataset_id
                               for x in gce_block_device_api_2.list_volumes()))
 
+    def test_list_live_nodes_pagination_and_removal(self):
+        """
+        list_live_nodes should be able to walk pages to get all live nodes and
+        should not have nodes after they are destroyed.
+        """
```

Comment:
:question: Why bother testing pagination here? I'd remove that part.
Seems like this'd be a useful `ICloudAPI` test in the future.
Other drivers probably don't / can't support the `stop_node` API so perhaps it should just rely on creating and terminating test nodes.